### PR TITLE
Admin Bar: Fix - "edit with elementor" button hidden in old wordpress versions.

### PR DESCRIPTION
**Pull request by Nevoss**
~Depends on: https://github.com/elementor/elementor/pull/12902~

### CODE REVIEW

**Review by matipojo — APPROVED** [review]

File: modules/admin-bar/module.php [hunk 1]
```diff
@@ -62,6 +65,15 @@ public function enqueue_scripts() {
 			true
 		);
 
+		// This is a core script of WordPress, it is not required to pass the 'ver' argument.
```
Comment (matipojo) [inline modules/admin-bar/module.php:68]: Test the order?

**Review by matipojo — CHANGES_REQUESTED** [review]

**Review by matipojo — DISMISSED** [review]

File: modules/admin-bar/assets/js/frontend/module.js [hunk 2]
```diff
@@ -134,4 +134,4 @@ class AdminBar extends elementorModules.ViewModule {
 	}
 }
 
-new AdminBar();
+document.addEventListener( 'DOMContentLoaded', () => new AdminBar() );
```
Comment (kobizz) [inline modules/admin-bar/assets/js/frontend/module.js:137]:
```suggestion
jQuery( () => new AdminBar() );
```

**Review by kobizz — CHANGES_REQUESTED** [review]

File: modules/admin-bar/assets/js/frontend/module.js [hunk 3]
```diff
@@ -134,4 +134,6 @@ class AdminBar extends elementorModules.ViewModule {
 	}
 }
 
-new AdminBar();
+// Must listen to 'DOMContentLoaded' event, to make sure this module will be executed BEFORE Wordpress admin bar.
+// (Wordpress admin bar script listen to the same event and then execute their code).
```
Comment (matipojo) [inline modules/admin-bar/assets/js/frontend/module.js:138]:
```suggestion
// (WordPress admin bar script is listening to the same event and then execute his code).
```

**Review by matipojo — CHANGES_REQUESTED** [review]

File: modules/admin-bar/assets/js/frontend/module.js (hunk 2 above)
Comment (matipojo) [reply modules/admin-bar/assets/js/frontend/module.js:139]: @kobizz see in the code comment.

**Review by matipojo — DISMISSED** [review]

File: modules/admin-bar/assets/js/frontend/module.js (hunk 2 above)
Comment (kobizz) [reply modules/admin-bar/assets/js/frontend/module.js:139]: jQuery relies on `DOMContentLoaded` event

**Review by kobizz — DISMISSED** [review]

**Review by matipojo — APPROVED** [review]

**Review by kobizz — APPROVED** [review]

**Review by KingYes — APPROVED** [review]

**Pull request merged**